ansible/roles/prosody/files/mod_muc_webhooks.lua

```diff
@@ -411,6 +424,26 @@ function handle_occupant_access(event, event_type)
             return
         end
 
+        -- lobby events
+        if is_lobby then
+            if final_event_type == PARTICIPANT_JOINED then
+                module:log("debug", "Occupant %s joined lobby room %s", occupant.jid, room.jid);
+                if main_room:get_affiliation(occupant.bare_jid) == 'owner' or occupant.role == "moderator" then
+                    table.insert(moderator_occupants_in_lobby, occupant.bare_jid);
+                    return;
+                end
+                participant_access_event["eventType"] = PARTICIPANT_JOINED_LOBBY;
+            elseif final_event_type == PARTICIPANT_LEFT then
+                module:log("debug", "Occupant %s left lobby room %s", occupant.jid, room.jid);
+                found, key = util.table_contains(moderator_occupants_in_lobby, occupant.bare_jid);
+                if found then
+                    table.remove(moderator_occupants_in_lobby, key);
+                    return;
+                end
```

Comment:
One small nit here. There is no point of table_contains and remove if found. You can call just remove which returns the element that was removed or nil if nothing was removed. 
so it can be 
```
if not table.remove(moderator_occupants_in_lobby, key) then
    participant_access_event["eventType"] = PARTICIPANT_LEFT_LOBBY;
end
```

Reply:
test it please.